### PR TITLE
Feat/setup dashboard cms

### PR DESCRIPTION
**Pull request by mmcode9407**

### CODE REVIEW

**Review by wawrzonnn — APPROVED** [review]
Good job! I think it's time to introduce global styles for our headers. We have several common classes like .wrapper, .headerContent, .logo, .title, and .frameButtons that we use in every header. It might be beneficial to consolidate them.

File: src/components/Headers/DashboardHeader/DashboardHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import { Button } from '@/components/Button/Button'
+import { MyProfileButton } from '@/components/MyProfileButton/MyProfileButton'
+import logo from '@/assets/images/logo.png'
+
+import styles from './DashboardHeader.module.scss'
+
+const DashboardHeader = async () => {
+  const session = await getServerSession(authOptions)
+
+  const profile = session
```
Comment (darionauta) [inline src/components/Headers/DashboardHeader/DashboardHeader.tsx:21]: I wonder if we could shorten this code a bit?
For example:
`const profile = session && await findUserByEmail(session.user.email)`
`if(!profile || !profile?.roles.includes(Role.MODERATOR)) redirect(AppRoutes.home)`

**Review by darionauta — APPROVED** [review]

File: app/(moderation)/dashboard/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  return (
+    <html lang="en">
+      <body className={ibm.className}>
```
Comment (Sumick) [inline app/(moderation)/dashboard/layout.tsx:22]: **nitpick:** This should not be a root layout, I will explain it in review summary

**Review by Sumick — CHANGES_REQUESTED** [review]
We need to move RootLayout to the root of `app` dir, based on documentation. HTML, body tag, and fonts should be declared only once there. https://nextjs.org/docs/app/api-reference/file-conventions/layout

Then we should create DashboardLayout component, which holds only DashboardHeader, Container, and Children. `AuthProvider` can be in RootLayout

**Review by Sumick — APPROVED** [review]
Looks really nice now! Good job 🙌

**Pull request merged**